stage23/drivers/vga_textmode.s2.c

```diff
@@ -139,6 +139,10 @@ void init_vga_textmode(size_t *_rows, size_t *_cols, bool managed) {
         current_video_mode = -1;
     }
 
+    if (!pmm_allocation_allowed()) {
+        front_buffer = back_buffer = video_mem;
```

Comment:
Wouldn't setting it to not ready cause nothing to be printed out?
It is obviously a very crude thing to do and does cause artifacts on the screen, but it is fine for panic.
What about setting it to not ready, unless the recursive panic flag is set, such that this never gets invoked under any other situation?
![image](https://user-images.githubusercontent.com/47392064/145674346-f4f38268-6eb3-4b5d-a694-95774389a702.png)

Reply:
Then a better idea would be, in `panic()` to, instead of `if (recursive)`, do `if (allocations_disallowed && term_backend == NOT_READY)` and avoid the recursive panic entirely anyways, since it can only really happen this way.